frontend/src/Movie/Index/Posters/MovieIndexPoster.js

```diff
@@ -127,6 +132,23 @@ class MovieIndexPoster extends Component {
       height: `${posterHeight}px`
     };
 
+    let releaseDate = '';
+    if (showReleaseDate) {
+      if (physicalRelease && digitalRelease) {
+        if (physicalRelease < digitalRelease) {
+          releaseDate = physicalRelease;
+        } else {
+          releaseDate = digitalRelease;
+        }
+      } else if (physicalRelease && !digitalRelease) {
+        releaseDate = physicalRelease;
+      } else if (digitalRelease && !physicalRelease) {
+        releaseDate = digitalRelease;
+      } else {
+        releaseDate = inCinemas;
+      }
+    }
```

Comment:
`    let releaseDate = '';
    if (showReleaseDate) {
	  releaseDate = inCinemas;
      if (physicalRelease && digitalRelease) {
	    releaseDate = (physicalRelease < digitalRelease) ? physicalRelease : digitalRelease;
      } else if (physicalRelease) {
        releaseDate = physicalRelease;
      } else if (digitalRelease) {
        releaseDate = digitalRelease;
	  }
    }`

I think that (or close to it, i didn't test it) would simplify it a little?

Reply:
That ternary operator was exactly the line I couldn't think of when I was writing it! I kept saying to myself that there was a different way to do it! I have updated that and the commit is incoming